src/pages/teachers/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug].tsx

```diff
@@ -90,16 +90,18 @@ export const getStaticProps: GetStaticProps<
           notFound: true,
         };
       }
-
-      const { videoTitle } = curriculumData;
-      if (videoTitle && !isSlugLegacy(programmeSlug)) {
+      const { videoTitle, transcriptSentences } = curriculumData;
+      if (videoTitle && !isSlugLegacy(programmeSlug) && !transcriptSentences) {
         // For new content we need to fetch the captions file from gCloud and parse the result to generate
         // the transcript sentences.
         const fileName = `${videoTitle}.vtt`;
         const transcript = await getCaptionsFromFile(fileName);
         if (transcript) {
           curriculumData.transcriptSentences = transcript;
         }
+      } else if (transcriptSentences && !Array.isArray(transcriptSentences)) {
+        const splitTranscript = transcriptSentences.split(/\r?\n/);
+        curriculumData.transcriptSentences = splitTranscript;
```

Comment:
I think we can update the type in the schema to remove `string` as a possibility if we're converting to array here

Reply:
Should I move this prior to parsing the data, as at that point it's still a string?